src/applications/mhv/secure-messaging/containers/LandingPageUnauth.jsx

```diff
@@ -34,15 +34,11 @@ const LandingPageUnauth = () => {
             <strong>My HealtheVet</strong> account. If you don’t have any of
             these accounts, you can create a free account now.
           </p>
-          {/* va-button component from design library has a bug which affects the styling */}

```

Comment:
Hello vfs-mhv-secure-messaging! 👋  We can no longer use the `.va-button-primary` Formation class so it has been replaced with `va-button` below but I noticed this comment here. I did not see any styling bug so I have removed this comment.

If there is still a bug with `va-button`, can you open an issue for the design system team?

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/new?assignees=&labels=platform-design-system-team%2C+bug&projects=&template=1_bug_template.md&title=

Reply:
Hi @jamigibbs Sorry, just seeing this now. It doesn't appear to affect us anymore, I'm ok with this change 👍🏻